bedrock/firefox/templates/firefox/testflight.html

```diff
@@ -1,131 +1,138 @@
- # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% extends "firefox/base/base-resp.html" %}
+{% extends "firefox/base/base-protocol.html" %}
+
+{% from "macros-protocol.html" import hero with context %}
 
 {% block page_title_prefix %}
-  {{ _('Firefox for iOS Beta') }} —
+{{ _('Firefox for iOS Beta') }} —
 {% endblock %}
 
 {% block page_title %}{{ _('TestFlight') }}{% endblock %}
 
 {% block page_desc %}
-  {{ _('Sign up to test pre-release beta versions of Firefox for iOS via Apple’s TestFlight program and help make our mobile browser for iPhone, iPad and iPod touch even better.') }}
+{{ _('Sign up to test pre-release beta versions of Firefox for iOS via Apple’s TestFlight program and help make our
+mobile browser for iPhone, iPad and iPod touch even better.') }}
 {% endblock %}
 
-{% block body_id %}firefox-ios-testflight{% endblock %}
-{% block body_class %}{% endblock %}
-
 {% block page_css %}
-  {{ css_bundle('firefox_ios_testflight') }}
+{{ css_bundle('firefox_ios_testflight') }}
 {% endblock %}
 
-{% block site_header_nav %}{% endblock %}
-{% block site_header_logo %}{% endblock %}
-
 {% block content %}
-<section id="intro">
-  <div class="container">
-    {# No more new logos! #}
-    {{ high_res_img('protocol/img/logos/firefox/browser/logo-lg.png', {'alt': 'Firefox for iOS', 'width': '142', 'height': '142', 'id': 'header-logo' }) }}
-
-    <header id="intro-header">
-      {# L10n: line break below for visual formatting only #}
-      <h1>{{ _('Help shape the future <br>of Firefox for iOS') }}</h1>
-
-      <h2>{{ _('Sign up to test pre-release versions to make Firefox even better.') }}</h2>
-    </header>
-
-    <div id="footer-email-errors" class="hidden newsletter-form-extra">
-      <ul class="errorlist"></ul>
-    </div>
-
-    <form id="newsletter-form" class="newsletter-form footer-newsletter-form" action="{{ url('newsletter.subscribe') }}" method="post" data-spinner-color="#fff">
-      {{ newsletter_form.newsletters|safe }}
-      {# test flight program is only available in english #}
-      <input type="hidden" name="lang" value="en">
-      <input type="hidden" name="source_url" value="{{ request.build_absolute_uri() }}">
-
-      <div id="form-fields">
-        <div id="email-submit-wrapper">
-          <div id="email-wrapper" class="field">
+<main role="main">
+  {% call hero(
+  title=_('Help shape the future <br>of Firefox for iOS'),
+  desc=_('Sign up to test pre-release versions to make Firefox even better.'),
+  class='mzp-t-firefox mzp-t-dark mzp-t-product-firefox',
+  include_cta=False,
+  heading_level=1) %}
+  {% endcall %}
+
+  <aside class="mzp-l-content">
+    <div class="mzp-c-newsletter">
+      <div class="mzp-c-newsletter-image">
+        {{ high_res_img('img/home/2018/newsletter-graphic.png', {'alt': ''}) }}
```

Comment:
This is a mozilla themes graphic - think we could either drop it, or find a more Firefox centric image?